muhammet-tarakci-comma-based.csl

```diff
@@ -0,0 +1,1300 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" and="symbol" delimiter-precedes-et-al="never" delimiter-precedes-last="never" page-range-format="chicago">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/#) -->
+  <info>
+    <title xml:lang="tr">Muhammet TARAKCI (Türkçe ve Virgül Ayrımlı Dipnot Sistemi)</title>
+    <title-short xml:lang="tr">muhammet-tarakci-comma-based</title-short>
+    <id>http://www.zotero.org/styles/muhammet-tarakci/comma-based</id>
+    <link href="http://www.zotero.org/styles/muhammet-tarakci" rel="self" xml:lang="tr"/>
+    <link href="http://muhammettarakci.blogspot.com.tr/2017/06/zotero-atf-sistemim.html" rel="documentation" xml:lang="tr"/>
+    <author>
+      <name>Muhammet Tarakçı</name>
+      <email>muhammettarakci@gmail.com</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="generic-base"/>
+    <summary>For Turkish authors. </summary>
+    <updated>2017-06-14T08:12:45+00:00</updated>
+  </info>
```

Comment:
it's missing the rights and hence not validating. Usually we have     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
 there.